src/isar/storage/utilities.py

```diff
@@ -14,12 +14,10 @@
 def construct_local_paths(
     inspection: Inspection, metadata: MissionMetadata
 ) -> Tuple[Path, Path]:
-    inspection_type: str = get_inspection_type(inspection=inspection)
-
     folder: Path = Path(str(metadata.mission_id))
     filename: str = get_filename(
         mission_id=metadata.mission_id,
-        inspection_type=inspection_type,
+        inspection_type=inspection.get_metadata_type_str(),
```

Comment:
Should this be metadata type or inspection type?

Reply:
yes! Good catch